config/ext_jenkins-infra-jobs.yaml

```diff
@@ -151,6 +151,9 @@ jobsDefinition:
     children:
       kubernetes-management:
         jenkinsfilePath: Jenkinsfile_updatecli
+      jenkins-infra:
+        name: Puppet (jenkins-infra)
```

Comment:
```suggestion
```

Reply:
the goal is to have the `Puppet` in  the custom name to avoid ambiguity with the jenkins-infra wording